Sources/Clients/ProfileStore/ProfileStore.swift

```diff
@@ -106,6 +108,13 @@ extension ProfileStore {
 		lens(\.profile)
 	}
 
+	/// A multicasting replaying async sequence of conflicting device the profile snapshot is used on.
+	public func usedOnOtherDevice() async -> AnyAsyncSequence<ProfileSnapshot.Header.UsedDeviceInfo> {
```

Comment:
async sequence that `BackupClient` expose, used in `Main` reducer to listen to events where Profile might be used on another device. Replacing old `throw error` based solution